libs/klv/src/main/java/org/codice/ddf/klv/KLV.java

```diff
@@ -0,0 +1,558 @@
+/**
```

Comment:
Yes, I changed a few things. I assume modifying and including public domain code in DDF is okay?